critiquebrainz/ws/review/views.py

```diff
@@ -283,10 +283,10 @@ def review_list_handler():
                 "id": "CC BY-NC-SA 3.0",
                 "info_url": "https:\/\/creativecommons.org\/licenses\/by-nc-sa\/3.0\/"
               },
+              "rating": 0,
```

Comment:
I understand that this will change in the future, but this has nothing to do with changes you are making in this pull request. Forget about pull request #125 for a moment and just focus on this one.

What is the work that is being done here? Changing usage of a term "rating" to "popularity". Is this directly related to the upcoming changes that you are working on to implement the rating functionality? Yes, but this is irrelevant in the context of this pull request. This is a separate part. Don't mix pull requests when they don't need to be, you'll only make it more confusing.